### PR TITLE
added the fav icon #41

### PR DESCRIPTION
**Pull request by Kratik1093**
added the new fav icon which matches theme of the food website 

issue no. #41

### CODE REVIEW

**Comment by AlfiyaSiddique** [comment]
@Kratik1093 
Can't see the favicon in the preview deploy:
image

If it's visible at your side can you share the image.

**Comment by Kratik1093** [comment]
Screenshot 2024-10-05 185500
its visible on my side

File: frontend/package.json [hunk 1]
```diff
@@ -33,5 +33,9 @@
     "postcss": "^8.4.32",
     "tailwindcss": "^3.4.0",
     "vite": "^5.0.8"
-  }
+  },
+  "main": "postcss.config.js",
+  "author": "",
+  "license": "ISC",
+  "description": ""
```
Comment (AlfiyaSiddique) [inline frontend/package.json:40]: @Kratik1093 why did you aded these lines here?

**Comment by AlfiyaSiddique** [comment]
Please reply to the comment and I am still not able to see the icon.
 But I will merge this PR and if it will not reflect in production then I will revert the PR again

File: frontend/package.json (hunk 1 above)
Comment (Kratik1093) [reply frontend/package.json:40]: to make this page run on my device i need to made these change otherwise the page was not starting on my device

**Comment by Kratik1093** [comment]
@AlfiyaSiddique I am also could understand why this is not working when deployed i checked the deployed code also but iant able to understand the problem

is the image in public section named tastytrial512.png , changes in html are visible to you in the deploy??

File: frontend/package.json (hunk 1 above)
Comment (AlfiyaSiddique) [reply frontend/package.json:40]: @Kratik1093 then please remove this while pushing the PR and use it for local only

Comment (Kratik1093) [reply frontend/package.json:40]: ok will generate another pr removing this part

Comment (AlfiyaSiddique) [reply frontend/package.json:40]: @Kratik1093 the above lines need to be removed to merge this Pr. Please remove it asap.

**Comment by AlfiyaSiddique** [comment]
@Kratik1093  please update the pr and add the chnages I have commented

**Comment by AlfiyaSiddique** [comment]
@Kratik1093 any updates?

**Comment by AlfiyaSiddique** [comment]
@Kratik1093 will be closing this PR and issue if no activity found by tomorrow.

**Pull request closed without merging**